crates/wasmtime/src/compile.rs

```diff
@@ -482,18 +486,17 @@ impl<'a> CompileInputs<'a> {
                 if translation.module.functions[func_index].is_escaping() {
                     self.push_input(move |compiler| {
                         let func_index = translation.module.func_index(def_func_index);
-                        let trampoline = compiler.compile_array_to_wasm_trampoline(
-                            translation,
-                            types,
-                            def_func_index,
-                        )?;
+                        let symbol = format!(
```

Comment:
Any concerns about the cost of doing these `format!`s unconditionally despite only using the result if there's an error?  If so, perhaps we could make this a lambda that we call from within the `with_context` closure.

Reply:
Nah they were gonna all happen anyway, I just moved them a bit earlier so cranelift failures now have the function name so I don't have to go hunting down which intrinsic is the one that needs updating

Reply:
Oops, nevermind, I didn't notice we use it in the non-error case.  Disregard!